packages/builder-vite/plugins/vue-docgen.ts

```diff
@@ -7,7 +7,7 @@ export function vueDocgen(vueVersion: 2 | 3): Plugin {
     name: 'vue-docgen',
 
     async transform(src: string, id: string) {
-      if (/\.(vue)$/.test(id)) {
+      if (/\.(vue)$/.test(id) && !/\.(stories)\.(vue)$/.test(id)) {
```

Comment:
We should be able to ditch the capturing groups here.

```suggestion
      if (/\.vue$/.test(id) && !/\.stories\.vue$/.test(id)) {
```